### PR TITLE
Development

### PR DESCRIPTION
**Pull request by Aaron-RN**

### CODE REVIEW

**Comment by Torres-ssf** [comment]
### Pull Request Under Review By Sérgio (TSE)

**Comment by Torres-ssf** [comment]
### Great work team 👏
But before merging to master there are somethings that we need to address:
 
 I notice that there 2 files with the names of each one of you. I must ask, you both worked on the same page? Microverse guidelines require that both of you work collaboratively in the same project. You can only work individually in individual projects. 

I only reviewed the 'Aaron.html' since its the one who matches more of the required guidelines from Microverse. Please rename it to 'index.html' and delete the other one. This can confuse the next TSE that going to review your project.

- [x] Add content to all container from the navbar (just like you did with books and events).

- [x] Add content to all main article(first section) and make it an article tag instead of div tag. Just like you did with 2 of them.

- [x] Add content to the last container of the third section and make it an article tag instead of div tag.

Please kindly make these changes and submit for code review again.

File: Aaron.html [hunk 1]
```diff
@@ -0,0 +1,262 @@
...
+    <body>
+        <header>
+            <h3 class="display-none">Smashing Magazine Heatmap</h3>
+            <nav>
+                <div class="gray-1 col-1-1 height-200">
+                    <div class="col-1-4 pad-0 margin-0">
+                        <div class="col-1-1 height-100"></div>
+                    </div>
+                    <div class="col-1-2 pad-0">
+                        <div class="col-1-5 height-100"></div>
```
Comment (Torres-ssf) [inline Aaron.html]: Add content to all containers

File: Aaron.html [hunk 2]
```diff
@@ -0,0 +1,262 @@
...
+                <article class="featured-article col-1-2 height-700 gray-3">
+                    <div class="flex-container alignitems-center flex-row text-white margin-b-20">
+                        <div class="imgblock-small rotate-20"></div>
+                        <div><p class="header3 font-Roboto">Ben Frain </p><span class="italic size-22">wrote</span></div>
+                    </div>
+                    <div class="text-align-left text-white featured-article-title">
+                        <h1>Designing And Building A Progressive Web Application Without A Framework (Part 2)</h1>
+                    </div>
+                </article>
+                <div class="col-1-2 height-700 gray-2"></div>
```
Comment (Torres-ssf) [inline Aaron.html]: Add content

File: Aaron.html [hunk 3]
```diff
@@ -0,0 +1,262 @@
...
+                <div class="col-1-2 height-700 gray-4"></div>
```
Comment (Torres-ssf) [inline Aaron.html]: Add content

**Review by Torres-ssf — CHANGES_REQUESTED** [review]

**Comment by Oghenebrume50** [comment]
Guys this is good, everything is working properly, you can submit this project and move on. :+1:

**Pull request merged**